OneCryptoPkg/OneCryptoBin/Support/OneCryptoMmBin.inf

```diff
@@ -0,0 +1,24 @@
+## @file
+#  This module provides OpenSSL Binary implementation.
+# TODO [ONE_CRYPTO] Generate this from the INF that builds the binary
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = OneCryptoMmBin
+  FILE_GUID                      = 76ABA88D-9D16-49A2-AA3A-DB6112FAC5CC
+  MODULE_TYPE                    = MM_STANDALONE
+  VERSION_STRING                 = 1.0
+  PI_SPECIFICATION_VERSION       = 0x00010032
+  ENTRY_POINT                    = MmEntry
+
+[Packages]
+  CryptoPkg/CryptoPkg.dec
```

Comment:
Doesn't need this right? For this INF I mean. Probably can drop the Protocols too, because that is only defined in mu_crypto_release which consumers won't be pulling.

Reply:
Necessary for TlsLib.h which defines TLS, we'll need to strongly control both